addon/appModules/notepad++/editWindow.py

```diff
@@ -143,10 +150,29 @@ def script_reportFindResult(self, gesture):
 			#Translators: Message shown when there are no more search results in this direction using the notepad++ find command.
 			speech.speakMessage(_("No more search results in this direction."))
 	
-	#Translators: when pressed, goes to    the Next search result in Notepad++
+	#Translators: when pressed, goes to	   the Next search result in Notepad++
 	script_reportFindResult.__doc__ = _("Queries the next or previous search result and speaks the selection and current line of it.")
 	script_reportFindResult.category = "Notepad++"
 
+	def script_htmlPreview(self, gesture):
+		from threading import Timer
+		repeatCount=scriptHandler.getLastScriptRepeatCount()
+		raw=self.windowText
```

Comment:
Have you specifically tested this with thousands of lines of text? The windowText I think might only be displayed text, it would probably be better to use a textInfo here, but I'm happy to let this pass if you can demonstrate it is working on a large document.